srcpkgs/qjackctl/template

```diff
@@ -14,6 +14,12 @@ homepage="http://qjackctl.sourceforge.net"
 distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.gz"
 checksum=5196c5c01b7948c1a8ca37cd3198a7f0fe095a99a34a67086abd3466855b4abd
 
+build_options="jack_session"
+build_options_default="jack_session"
+desc_option_jack_session="Enable support for the deprecated Jack Session API"
+
+configure_args="$(vopt_enable jack_session)"
```

Comment:
I think you need to use a dash instead of an underscore for the build option.

Reply:
Using a dash doesn't work since `build_option_jack-session` is an invalid variable name.
This is why I initially used an if statement checking the option, but I tested and verified that using `jack_session` works properly.

Reply:
Changed to `$(vopt_enable jack_session jack-session)`, thanks!